tutorials/ner_pytorch_medical/project.yml

```diff
@@ -0,0 +1,109 @@
+title: "Detecting entities in Medical Records with PyTorch"
+description: "This project uses the [i2b2 (n2c2) 2011 Challenge Dataset](https://portal.dbmi.hms.harvard.edu/projects/n2c2-nlp/) to bootstrap a PyTorch NER model to detect entities in Medical Records. It also demonstrates how to anonymize medical records for annotators in [Prodigy](https://prodi.gy)."
+# Variables can be referenced across the project.yml using ${vars.var_name}
+vars:
+  spacy_config: "spacy_config.cfg"
+  config: "config.cfg"
+  config_trf: "config_trf.cfg"
+  name: "ner_pytorch_medical"
+  version: "0.0.0"
+  train: "train"
+  dev: "dev"
+  test: "test"
+
+# These are the directories that the project needs. The project CLI will make
+# sure that they always exist.
+directories: ["assets", "training", "configs", "scripts", "corpus", "packages"]
+
+# Workflows are sequences of commands (see below) executed in order. You can
+# run them via "spacy project run [workflow]". If a commands's inputs/outputs
+# haven't changed, it won't be re-run.
+workflows:
+  all:
+    - preprocess
+    - train
+    - evaluate
+
+# Project commands, specified in a style similar to CI config files (e.g. Azure
+# pipelines). The name is the command name that lets you trigger the command
+# via "spacy project run [command] [path]". The help message is optional and
+# shown when executing "spacy project run [optional command] [path] --help".
+commands:
+  - name: "preprocess"
+    help: "Convert the data to spaCy's binary format"
+    script:
+      - "python scripts/preprocess.py assets/n2c2_2011 corpus"
+    deps:
+      - "assets/n2c2_2011/i2b2_Beth_Train_Release.tar.gz"
+      - "assets/n2c2_2011/i2b2_Partners_Train_Release.tar.gz"
+      - "assets/n2c2_2011/Task_1C.zip"
```

Comment:
I've found it useful to also include the Python script in the `deps` - then it'll definitely be rerun when you change the script. I think otherwise it'll be skipped if the `assets` file remained the same.

Reply:
That's certainly more elegant than me commenting out the `deps` lines while developing hahaha

Reply:
btw, for future reference, you can also use `spacy project run ... --force` ;-)